### ISSUE TITLE
AOTI tutorial `CUDA_HOME`

### ISSUE THREAD

**Issue by bhack** [pytorch/tutorials#3153]
### 📚 The doc issue

I am trying to follow https://pytorch.org/tutorials/recipes/torch_export_aoti_python.html but it seems it doesn't work on our nightly runtime images cause missing `CUDA_HOME`.

cc @svekars @brycebortree @sekyondaMeta @ezyang @chauhang @penguinwu @avikchaudhuri @gmagogsfm @zhxchen17 @tugsbayasgalan @angelayi @suo @ydwu4 @desertfire @chenyang78

**Comment by bhack** [comment]
Any update?

**Comment by agunapal** [comment]
@bhack i will update the tutorial for PT 2.6

**Comment by agunapal** [comment]
Yes, I will mention in the updated tutorial that you need a devel image to create .pt2 package
